homeassistant/components/netgear_lte/binary_sensor.py

```diff
@@ -1,39 +1,44 @@
 """Support for Netgear LTE binary sensors."""
 from __future__ import annotations
 
-from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
```

Comment:
I think `BinarySensorEntityDescription` could be implemented in a separate PR (preliminary or follow-up)